src/cli/main.py

```diff
@@ -57,11 +58,7 @@ def execute_command(cmd: str):
 
 
 def main():
-    # TODO: Issue #20
-    project_home = os.path.dirname(os.path.abspath(__file__)) + "/.."
-    os.chdir(project_home)
-
-    _setup_logging()
+    _setup_logging(file=False)
```

Comment:
I agree, however I didn't make any changes to the default here. This is the behavior we had before, and that is a seperate issue.